custom/icds_reports/reports/mwcd_indicators.py

```diff
@@ -0,0 +1,80 @@
+from datetime import date, timedelta
+from custom.icds_reports.models.views import MWCDReportView
+from django.db.models import Sum
+
+
+def get_implementation_status_data():
+    latest_month_available = (date.today() - timedelta(days=1)).replace(day=1)
+
+    implementation_stat = MWCDReportView.objects.filter(month=latest_month_available).values(
+        'state_name',
+        'state_id',
+        'num_launched_awcs',
+        'num_launched_districts',
+        'num_launched_states',
+        'num_ls_launched',
+        'awc_with_gm_devices',
+        'cases_household',
+        'total_mothers',
+        'cases_child_health'
+    )
+
+    total_count = {
+        "nation_code": 0,
+        'num_launched_awcs': 0,
+        'num_launched_districts': 0,
+        'num_launched_states': 0,
+        'num_ls_launched': 0,
+        'awc_with_gm_devices': 0,
+        'cases_household': 0,
+        'total_mothers': 0,
+        'cases_child_health': 0
+    }
+
+    for record in implementation_stat:
```

Comment:
Is this not doable at the ORM level? Its just a simple sum

Reply:
I did not wanted to make another request of SUM to DB especially when the data is already fetched. I think Even when the query time is fast. It was not making sense to send another query to db.
@calellowitz

Reply:
I see, i didnt realize you were returning the whole list as well. thats probably fine

Reply:
@calellowitz  was that an approval?